### PR TITLE
feat: Adding Next.js app following the agreed directory structure

### PR DESCRIPTION
**Pull request by PeterMarinov**
# Adding Next.js app following the agreed directory structure

## Description
We will use `apps/endatix-hub` to setup the Endatix management UI.

To run the app:
1. cd into `apps/endatix-app`
2. run `npm install`
3. run `npm run dev`

## Related Issues
unblocks #84 

## Type of Change
- [x] New feature (non-breaking change which adds functionality)

### CODE REVIEW

File: apps/endatix-hub/services/sitemap-service.ts [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+    if (!excludeSettings){
+      sitemapList.push(sitemap.Settings);
+    }
+
+    return sitemapList;
+  }
+
+  public static getLogo(): INavItem {
+    const logoNavItem: INavItem = {
+      path: "https://surveysocket.com",
```
Comment (dspetrov) [inline apps/endatix-hub/services/sitemap-service.ts:35]: This and the below are left with the old name.

**Review by dspetrov — CHANGES_REQUESTED** [review]
LGTM, I've added a comment considering an old name left.

File: apps/endatix-hub/services/sitemap-service.ts (hunk 1 above)
Comment (PeterMarinov) [reply apps/endatix-hub/services/sitemap-service.ts:35]: Great for spotting this. Let me fix in the next commit

**Comment by PeterMarinov** [comment]
@dspetrov this is done. Also added the auth WIP piece. Step 1 to 3 are done, but stopping here to keep scope clean:
image

**Review by dspetrov — APPROVED** [review]

**Pull request merged**